tests/Sorry.Analyzers.Tests/RR1337IfStatementInTestMethodTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace Sorry.Analyzers.Tests
+{
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using RoslynTestKit;
+    using Xunit;
+
+    public class RR1337IfStatementInTestMethodTests : AnalyzerTestFixture
```

Comment:
Please rename testclass appropriatelly

Reply:
I renamed it. Is the current name appropriate enough?

Reply:
the most appropriatest ever